src/components/user-profile/index.js

```diff
@@ -1,30 +1,23 @@
 import { useState, useEffect } from 'react';
 import { userContext } from '@store/user/user-context';
 import classNames from '@components/user-profile/user-profile.module.scss';
-import { getUserSelf } from '../../helper-functions/action-handlers';
 
 const UserProfile = () => {
   const [isLoading, setIsLoading] = useState(true);
   const {
     user,
-    setUser,
     isSuperUser,
-    setIsSuperUser,
     isSuperUserMode,
     setIsSuperUserMode,
+    makeApiCallToVerifySuperUser,
```

Comment:
Nit: This can be a shorter verb "verifySuperUser". We don't need to write the imperative steps of _how_ we are doing it.
Otherwise, many functions will start looking like this:
- callFunctionFromClassWalletToCheckForAmountRemaining
- checkIfEmailIsValidByMatchingRegularExpressionForEmail

Reply:
Yeah Right. Thanks 🙂